port/unix/omrsyslog.c

```diff
@@ -153,6 +154,11 @@ get_conv_txt_num_bytes(struct OMRPortLibrary *portLibrary, const char *message)
 static void
 syslog_write(struct OMRPortLibrary *portLibrary, const char *buf, int priority)
 {
+#if defined(OMRZTPF)
+	sprintf(buf, "%s", buf);
```

Comment:
@keithc-ca 
Thanks, I will put it on my radar.  
I just undid the change in our vm28 build environment and it compiles cleanly.  
Not sure why we had that update to begin with.  I need discuss it with Pete L. to see if we can remember why we made this update.   If we can't remember, I'll undo it in our next pull request coming shortly with updates we need for omrsysinfo.  This update only affects the z/TPF platform and shouldn't have affected anyone else.